docs/up-logs/sessions/v1.md

```diff
@@ -11,8 +11,27 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total sessions for the landing page of a work, as determined by processing the Access Logs of the website. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity; if a single user keeps interacting with the website within this time frame, multiple visits to the same work will be counted as one session.
+Total views of the landing page of a work, as determined by processing the access logs (i.e. webserver logs) of the website. Multiple views of the same landing page by the same user within a rolling 30 minute window are counted as one.
```

Comment:
This is phrased differently from `up-ga/sessions`... can we use the same terminology?